db/revtree.go

```diff
@@ -381,6 +377,20 @@ func ParseRevisions(body Body) []string {
 	return result
 }
 
+// Splits out the "start" and "ids" properties from encoded revision list
+func splitRevisionList(revisions Body) (int, []string) {
+	start, ok := base.ToInt64(revisions["start"])
+	digests, _ := GetStringArrayProperty(revisions, "ids")
```

Comment:
It's going to detect an error because `digests` will be nil. The details don't matter at this level because the functions that call this aren't returning specific errors, just success/failure. (If someone's screwed up this data they'll end up getting a 400 status back.)